tests/unit/models/transactions/test_xchain_modify_bridge.py

```diff
@@ -0,0 +1,113 @@
+from unittest import TestCase
+
+from xrpl.models import (
+    XRP,
+    IssuedCurrency,
+    XChainBridge,
+    XChainModifyBridge,
+    XRPLModelException,
+)
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
```

Comment:
Why do these variables have underscores on them? They seem to be used.

Reply:
Underscored variables are specific to the file. Prepending an underscore to a variable name makes it a "private" variable, so you're indicating that it shouldn't be imported out of this file. In test files like this it doesn't really matter much, but it's still good practice.